ew/static/mutations.py

```diff
@@ -537,6 +547,104 @@
         tier=4,
         str_transplant="Dr. Dusttrap walks to the back of clinic and returns with a pickled punk. She unscrews the lid and carefully removes the preserved human fetus with tongs. \"This operation's a bit archaic, but it's not like you've got any real qualms, right?\" She instructs you to lie down, and holding your eyes open, Dr Dusttrap carefully positions the fetus above you horizontally. She lowers it inches from your head before giving it a harsh squeeze. Gallons of fetal juice pour directly into your eyes. Dusttrap puts the fetus back in the jar, then returns it to the back of the clinic. By the time she returns, your eyes have become horribly infected. You have developed the mutation **Stink Eye**. When surveying, the amount of slime on the ground is shown, along with 4 items starting with the lowest IDs.",
     ),
+    EwMutationFlavor(
+        id_mutation=ewcfg.mutation_id_monplanto,
+        str_name="Monplanto",
+        alias=['mp', 'mon', 'plant'],
+        str_describe_self="You've harnessed the power of the sun due to **Monplanto**.",
+        str_describe_other="They've harnessed the power of the sun due to **Monplanto**.",
+        str_acquire="Wow, it sure is sunny out today, huh? You feel solar radiation frying your skin at such a deep level it begins to affect your DNA. For now you are a being of the sun, and can truly bask in its intense rays. You have developed the mutation **Monplanto**. Wilted crops can be reaped normally, and during sunny weather you gain passive hunger regeneration.",
+        tier=1,
+        incompatible = {
+            ewcfg.mutation_id_airlock : '"Nope, you already have that mutation. It\'s even got a bunch extra tacked on."'
+        },
+        str_transplant="Dr. Dusttrap guides you towards a tanning bed, putting it at super-duper-mega-ultra intensity. Your skin sloughs off from the UV rays, but the raw flesh below is infused with the blue tanning light. You have developed the mutation **Monplanto**. Wilted crops can be reaped normally, and during sunny weather you gain passive hunger regeneration.",
+    ),
+    EwMutationFlavor(
+        id_mutation=ewcfg.mutation_id_foghorn,
+        str_name="Fog Horn",
+        alias=['fog', 'foghorn', 'horn', 'fh'],
+        str_describe_self="You can cut through fog due to **Fog Horn**.",
+        str_describe_other="They can cut through fog due to **Fog Horn**.",
+        str_acquire="You gain a splitting headache. A headdache *so splitting* that your head begins splitting in half. A massive horn burgeons out of your cranium. It's not that bad though; you're probably ten times as aerodynamic now. You have developed the mutation **Fog Horn**. During foggy weather, gain an increased critical hit chance.",
```

Comment:
consider the possibility of the horn being like an aiming reticle, thereby increasing your crit chance